src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/SqliteDatabaseProvider.cs

```diff
@@ -16,6 +17,7 @@ namespace Jellyfin.Database.Providers.Sqlite;
 [JellyfinDatabaseProviderKey("Jellyfin-SQLite")]
 public sealed class SqliteDatabaseProvider : IJellyfinDatabaseProvider
 {
+    private const string BACKUPFOLDERNAME = "SQLiteBackups";
```

Comment:
```suggestion
    private const string BackupFolderName= "SQLiteBackups";
```

Surprised the analyzer didn't flag the full caps 😆